### PR TITLE
Task/update bookings view

### PR DESCRIPTION
**Pull request by learner1unknown**
# Created bookings view

## Has this been tested? How?

locally

## Types of change

- [ ] New feature (change which adds functionality)

- [ ] Bug fix (change which fixes an issue)

- [x] Improvement/refactoring (change that doesn't add any features but makes things better)

- [x] Styling (style changes that don't affect functionality)

## Checklist before submitting

- [x] Satisfies card on Trello

- [ ] Has had a another team mate review your design.

- [x] No known defects

- [ ] Appropriate tests completed (e.g. core functionality, edge cases)

- [ ] Code has been formatted with nicely and contains proper indentation.

- [ ] Code comments as appropriate

- [x] `master` branch has been merged into the feature branch and tested (if any newer commits on `master`)

### CODE REVIEW

File: app/views/bookings/index.html.erb [hunk 1]
```diff
@@ -1,76 +1,70 @@
...
+          <%= link_to "Delete", booking_path(your_booking), class: "btn btn-danger text-white" ,data: { turbo_method: :delete, turbo_confirm: "Are you sure?" } %>
         </div>
       </div>
+    </div>
     <% end %>
   </div>
+</div>
 
-    <div class="row">
+<div class="container mt-3">
```
Comment (drenetic) [inline app/views/bookings/index.html.erb:37]: I'd give this a little more headroom: mt-5

**Review by drenetic — APPROVED** [review]
